### DISCUSSION TITLE
Cloudflared: "waiting for login"

### DISCUSSION THREAD

**Discussion by gpjob** [indomus/forum#2971] [HUB: Home Assistant]
Ciao, seguito alla lettera la guida "https://indomus.it/guide/collegarsi-da-remoto-a-home-assistant-via-reti-provider-tradizionali-o-che-usino-nat-via-cloudflare/", in particolare quella per "Home Assistant OS / Supervised", ho registrato il sito, configurato Cloudflared, fatto partire tutto e ha funzionato tutto tranne che l'ultimo passaggio, in base al quale il registro di Cloudflared mi indica: 
                                                               "2024-03-11T16:28:07Z INF Waiting for login...".
Ringrazio anticipatamente qualsiasi aiuto

**Comment by snakuzzo** [comment]
https://community.cloudflare.com/t/cloudflared-argo-tunnel-login-problem/86163
